client/app/layouts/main.html

```diff
@@ -5,4 +5,14 @@
        class="md-whiteframe-2dp main-panel">
     <ui-view></ui-view>
   </div>
-</div>
\ No newline at end of file
+</div>
+<p class="footer">
+  made with &#9829; by <a href="https://yunity.org">yunity</a> |
+  contribute on <a href="https://github.com/yunity/foodsaving-frontend"><i class="fa fa-github"></i> Github</a> |
```

Comment:
had troubles with angular-translate and HTML in the strings and it has not such an high priority, so i decided to leave it like that.
but feel free to change it if you want ;)